### PR TITLE
Service Instances => Service Handler Instances; Remove reference to runnable

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-388

Bamboo : http://builds.cask.co/browse/CDAP-DUT911

### CODE REVIEW

File: cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java [hunk 1]
```diff
@@ -389,6 +391,27 @@ public void setProcedureInstances(String appId, String procedureId, int instance
   }
 
   /**
+   * Gets the number of instances of a service handler.
```
Comment (ParthGandhi) [inline cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java]: I think now that they're separated we should just refer to them as `Service` and not "service handler". This is a general comment for all javadocs/documentation.

The reason is that you can't set instances for individual handlers, but only for the service as a whole. Also, now that workers have been moved out we don't need to distinguish the types.

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java [hunk 2]
```diff
@@ -1079,6 +1079,18 @@ public void getWorkflowSchedules(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Return the number of instances of a service handler.
```
Comment (ParthGandhi) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java]: Same here.

**Comment by ParthGandhi** [comment]
Just one comment. Otherwise LGTM.

**Comment by gokulavasan** [comment]
@ParthGandhi Addressed your comments. Will merge after build passes.

**Pull request merged**